docs/dev/index.rst

```diff
@@ -147,3 +162,44 @@ Contribute additional tests
 Additional test files should be added to `this repository <sisl-files_>`_.
 Please follow the guidelines there, or open up an issue at that repository
 for specific details.
+
+
+Contribute to the docs
+----------------------
+
+To contribute to the documentation one needs to install `pandoc` first (see point
+**1.** in **Summary of development process** above). Then follow these steps:
+
+1. Sitting inside the `sisl` tree, install the `sisl` documentation via:
+
+   .. code:: bash
+
+      pip install -e .[docs]
+
+2. Download tutorial files outside the `sisl` repository:
+
+   .. code:: bash
+
+      git submodule init
+      git submodule update
+
+3. **OPTIONAL** : If your are not contributing specifically to the notebooks,
+   you may consider deactivating their compilation by commenting out (or eliminating)
+   this line ``"nbsphinx",`` in ``sisl/docs/conf.py`` file. This will enormously speed up
+   building times (see below).
+
+4. Within the docs folder (``sisl/docs``) do:
+
+   .. code:: bash
+
+      make html
+
+   This will build the documentation in the ``sisl/docs/build/html`` folder. Open any
+   **.html** file sitting there in your browser to visualize the built docs.
+
+5. Now you can contribute to the documentation by modifying any **.rst** file (reStructuredText
+   , or reST) sitting in ``sisl/docs``. Then build again (``make html``) and check your changes
+   in the browser.
```

Comment:
You can contribute in any way :) Perhaps we could phrase it like: "The easiest thing that you can do is to modify one of the rst files"